staging/cse/windows/containerdtemplate.toml

```diff
@@ -46,6 +46,8 @@ state = "C:\\ProgramData\\containerd\\state"
       [plugins.cri.containerd.runtimes]
         [plugins.cri.containerd.runtimes.runhcs-wcow-process]
           runtime_type = "io.containerd.runhcs.v1"
+          container_annotations = ["io.microsoft.container.processdumplocation"], ["io.microsoft.container.processdumpcontainersubdirectory"], ["io.microsoft.wcow.processdumptype"], ["io.microsoft.wcow.processdumpcount"]
```

Comment:
We can update container_annotations and pod_annotations to include only the 3 following strings:

"io.microsoft.container.processdumplocation"
"io.microsoft.wcow.processdumptype"
"io.microsoft.wcow.processdumpcount"

reference: 
https://github.com/microsoft/hcsshim/blob/a27618474e0bc32622fc2890bceb77161f7e90fc/pkg/annotations/annotations.go#L233
https://github.com/microsoft/hcsshim/pull/1929

Reply:
@kiashok can we add `["io.microsoft.container.processdumplocation"]` in `pod_annotations`?